docs/spec_draft.md

```diff
@@ -82,6 +82,17 @@ The specification of an op comprises of the following components (in the order
   * **Constraints** Type constraints on the operand(s), result(s).
   * **Examples** Examples demonstrating the working of the op.
 
+
+## Index of documented ops
```

Comment:
Let's just calls this "Ops" to match other similar headers which are called "Types", "Programs", etc?